DTML.EduBot/Dialogs/GreetingDialog.cs

```diff
@@ -9,13 +9,14 @@
 
 namespace DTML.EduBot.Dialogs
 {
-    public partial class ChitChatDialog : QnaLuisDialog<object>
+    public partial class NoneDialog : QnaLuisDialog<object>
```

Comment:
Why did we rename this? did we change the functionality from chit chat, to handling only "None" intents?